sematic/ui/packages/main/src/pipelines/BasicMetricsPanel.tsx

```diff
@@ -0,0 +1,88 @@
+import { Run } from "@sematic/common/src/Models";
+import { usePipelineRunContext } from "src/hooks/pipelineHooks";
+import useBasicMetrics from "src/hooks/metricsHooks";
+import Loading from "src/components/Loading";
+import { Alert, Box, Grid, Table, TableBody, TableCell, TableRow, Tooltip, Typography, useTheme } from "@mui/material";
+import { useMemo } from "react";
+import {durationSecondsToString} from "src/utils";
+import CalculatorPath from "src/components/CalculatorPath";
+import HelpIcon from "@mui/icons-material/Help";
+import { BasicMetricsPayload } from "src/Payloads";
+
+function TopMetric(props: {value: string, label: string, help?: string}) {
+  const {value, label, help } = props;
+  const theme = useTheme();
+
+return <Box sx={{textAlign: "center", p: 10}}>
```

Comment:
I prefer styled components because of that:
1. separates styling logic from structure logic 
2. I can use CSS syntax instead of JS syntax (`margin-top` vs `marginTop`), I can copy&paste things from browser dev tools directly as source code.

Eg. 

```jsx
import styled from "@emotion/styled" // <-- notice the import package name, don't use @mui, which has bugs

const StyledBox = styled(Box)`
    text-align: center;    
`
```

one can even nest styles, so we only need to have one styled component. Eg:

```jsx
const StyledBox = styled(Box)`
    text-align: center;
    
    & .BigText {
       font-size: 70
       font-weight: 500;
       lineHeight: 1
    }
`
```

then we only need to write 
```jsx
<Typography className={"BigText"}>{value}</Typography>
```
instead of 
```jsx
<Typography sx={{fontSize: 70, fontWeight: 500, lineHeight: 1}}>{value}</Typography>
```

we can also use

```jsx
const StyledBox = styled(Box)`
    text-align: center;
    
    & .MuiTypography-root {
       font-size: 70
       font-weight: 500;
       lineHeight: 1
    }
`
```
to address all nested \<Typography /> components